BasicAuthontificator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using APIAutomation;
+using NUnit.Framework;
+using RestSharp;
+using RestSharp.Authenticators;
+
+namespace APIAutomation
+{
+    public class BasicAuthenticator : AuthenticatorBase
+    {
+        readonly string _baseUrl;
+        readonly string _clientUsername;
+        readonly string _clientPassword;
+        readonly string _scope;
+
+        public BasicAuthenticator(string baseUrl, string clientUsername, string clientPassword, string scope) : base("")
```

Comment:
I would suggest to use scope variable with Enum type to set restrictions for this parameter. In theory string can apply any value besides "read" and "write", and it can lead to unexpected flow

Reply:
Ok, it is done